keyboards/tada68/keymaps/mekberg/rules.mk

```diff
@@ -0,0 +1,18 @@
+# Build Options
```

Comment:
This file can be condensed down to:

```make
MOUSEKEY_ENABLE = yes
CONSOLE_ENABLE = no
```

Reply:
This is the exact same as the default rules.mk file, so perhaps that should also be adjusted in that case? If nothing else, the actual hardcoded defaults should be made more clear, since there are several other options being enabled in both files, but I assume from your comment they are enabled by default anyway and thus don't need to be in this file regardless?

Reply:
Basically, stuff in the keymap level rules.mk overrides the keyboard-level one, so specifying eg. `BOOTMAGIC_ENABLE = no` here is redundant, in the same way you (usually) don't need to restate the MCU and bootloader. The above two lines are the only differences between this file and the keyboard one, so that's all that needs to go in it.

Reply:
Fair enough, I'll take a look and update the file.

Reply:
Took out redundancies and actually disabled mousekey as well since I don't even use it.